Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasi"
-version = "0.7.0"
+version = "0.8.0+wasi-snapshot-preview1"
```

Comment:
The actual wasi string has underscores rather than dashes; would it make sense to match that?

Reply:
I initially thought the same yeah but according to https://semver.org/ it says

```
<non-digit> ::= <letter>
              | "-"
```

which the `semver` crate in Rust follows so it doesn't allow underscores :(